client/gefyra/cluster/resources.py

```diff
@@ -159,6 +160,33 @@ def create_operator_deployment(
     return deployment
 
 
+def _check_pod_for_command(pod: V1Pod, index=0):
+    containers: list[V1Container] = pod.spec.containers
+    if not len(containers):
+        raise RuntimeError(f"No container available in pod {pod.metadata.name}.")
+
+    ALLOWED_COMMANDS = ["sh", "bash", "zsh", "ash", "/entrypoint.sh"]
+
+    if (
+        containers[index].command
+        and containers[index].command[0] not in ALLOWED_COMMANDS
+    ):
+        raise RuntimeError(
+            f"Cannot bridge pod {pod.metadata.name} since it has a `command` defined."
+        )
+
+
+def check_pod_valid_for_bridge(
+    config: ClientConfiguration, pod_name: str, namespace: str
+):
+    pod = config.K8S_CORE_API.read_namespaced_pod(
+        name=pod_name,
+        namespace=namespace,
+    )
+
+    _check_pod_for_command(pod)
```

Comment:
At this location, the information about the container I actually want to bridge is missing.